.github/workflows/test.yml

```diff
@@ -172,36 +169,30 @@ jobs:
       - name: Prepare the build
         run: test-drupal-project prepare_build
 
-      - name: Use Thunder 2 lockfile
+      - name: Use Thunder 3.4 lockfile
         run: |
           touch /tmp/test/thunder/finished-stages/build
-          cp ${GITHUB_WORKSPACE}/tests/fixtures/thunder2.composer.lock composer.lock
+          cp ${GITHUB_WORKSPACE}/tests/fixtures/thunder3-4.composer.lock composer.lock
```

Comment:
I guess we have to rename that file now to thunder3-6.composer.lock...

Reply:
That means Thunder 3.4 ;-)

Reply:
Ahh!